apps/channels/tests/message_examples/sureadhere_messages.py

```diff
@@ -0,0 +1,6 @@
+def inbound_message():
+    return {"message_text": "Hi", "patient_id": 6225}
+
+
+def outbound_message():
+    return {"patient_Id": 6225, "message_Body": "Hello, I am a chatbot, how can I help?"}
```

Comment:
Just noting that it's a bit odd that the inbound / outbound fields are different:
```diff
- message_text
+ message_Body
- patient_id
+ patient_Id
```

Reply:
No need to change it. I was just making sure this wasn't a typo or something.